src/api/cvc4cpp.cpp

```diff
@@ -2213,5 +2213,120 @@ std::vector<Expr> Solver::termVectorToExprs(
   return res;
 }
 
+/* Create operator terms                                                      */
+/* -------------------------------------------------------------------------- */
+
+OpTerm Solver::mkOpTerm(Kind kind, Kind k)
+{
+  // CHECK: kind == CHAIN_OP
+  return d_exprMgr->mkConst(CVC4::Chain(extToIntKind(k)));
+}
+
+OpTerm Solver::mkOpTerm(Kind kind, const std::string& arg)
+{
+  // CHECK:
+  // kind == RECORD_UPDATE_OP
+  return d_exprMgr->mkConst(CVC4::RecordUpdate(arg));
+}
+
+OpTerm Solver::mkOpTerm(Kind kind, uint32_t arg)
+{
+  OpTerm res;
+  if (kind == DIVISIBLE_OP)
+  {
+    res = d_exprMgr->mkConst(CVC4::Divisible(arg));
+  }
+  else if (kind == BITVECTOR_REPEAT_OP)
+  {
+    res = d_exprMgr->mkConst(CVC4::BitVectorRepeat(arg));
+  }
+  else if (kind == BITVECTOR_ZERO_EXTEND_OP)
+  {
+    res = d_exprMgr->mkConst(CVC4::BitVectorZeroExtend(arg));
+  }
+  else if (kind == BITVECTOR_SIGN_EXTEND_OP)
+  {
+    res = d_exprMgr->mkConst(CVC4::BitVectorSignExtend(arg));
+  }
+  else if (kind == BITVECTOR_ROTATE_LEFT_OP)
+  {
+    res = d_exprMgr->mkConst(CVC4::BitVectorRotateLeft(arg));
+  }
+  else if (kind == BITVECTOR_ROTATE_RIGHT_OP)
+  {
+    res = d_exprMgr->mkConst(CVC4::BitVectorRotateRight(arg));
+  }
+  else if (kind == INT_TO_BITVECTOR_OP)
+  {
+    res = d_exprMgr->mkConst(CVC4::IntToBitVector(arg));
+  }
+  else if (kind == FLOATINGPOINT_TO_UBV_OP)
+  {
+    res = d_exprMgr->mkConst(CVC4::FloatingPointToUBV(arg));
+  }
+  else if (kind == FLOATINGPOINT_TO_UBV_TOTAL_OP)
```

Comment:
This is somewhat orthogonal to this PR but we should check if users are supposed to be able to create `FLOATINGPOINT_TO_UBV_TOTAL_OP` and `FLOATINGPOINT_TO_SBV_TOTAL_OP` operators.

Reply:
According to @martin-cs those are internal. I'll remove in a separate PR.